package-lock.json

```diff
@@ -5472,59 +5473,6 @@
         "url": "https://opencollective.com/popperjs"
       }
     },
-    "node_modules/@remix-run/react": {
```

Comment:
@nikosdouvlis From a simple installation this got removed, is there any step I skipped ?

Reply:
This is a peer dep of clerk/remix: https://github.com/clerkinc/javascript/blob/41b61adc8d70b39b13123933a37946b8fce097d2/packages/remix/package.json#L54

Let's now lose time, feel free to go ahead and merge onto `next` and we'll fix it there directly if needed